File: code-examples/scala/collation/collection-collation.scala
// Import necessary MongoDB classes
import org.mongodb.scala._
import org.mongodb.scala.model._
import org.mongodb.scala.model.CreateCollectionOptions

// Instantiate the MongoClient and get the database
val mongoClient: MongoClient = MongoClient()
val database: MongoDatabase = mongoClient.getDatabase("mydb")

// Create the "souvenirs" collection with a French Canadian collation
database.createCollection(
  "souvenirs",
  CreateCollectionOptions().collation(Collation.builder().locale("fr_CA").build())
)

File: code-examples/php/collation/collection-collation.php
<?php
// Connection parameters
$manager = new MongoDB\Driver\Manager("mongodb://localhost:27017");

// French Canadian collation
$collation = ['locale' => 'fr_CA'];

// Create the "souvenirs" collection with the French Canadian collation
$createCommand = new MongoDB\Driver\Command([
    'create' => 'souvenirs',
    'collation' => $collation
]);

// Execute the command
$manager->executeCommand('mydb', $createCommand);
?>
